# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1300419577 25200
#      Thu Mar 17 20:39:37 2011 -0700
# Node ID 6a50edfb186beba1ec50aecac0a37e8cf5d48701
# Parent  77b14e634166d5d3951fca61b18213c82faf41aa
Add fmod, octave_tmp_file_name to list of exceptions not needing a DOCSTRING.

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,14 @@
+2010-03-17  Rik  <octave@nomad.inbox5.com>
+
+	* interpreter/doccheck/mk_undocumented_list: Add fmod,
+	octave_tmp_file_name to list of exceptions which are already
+	documented.
+
 2011-03-17  Iain Murray  <iain@iainmurray.net>
 
 	* interpreter/system.txi: Remove nprocs and nprocs_conf functions
 	from documentation and replace with nproc.
 
 2010-03-16  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/system.txi: Add nproc and nproc_conf functions to
diff --git a/doc/interpreter/doccheck/mk_undocumented_list b/doc/interpreter/doccheck/mk_undocumented_list
--- a/doc/interpreter/doccheck/mk_undocumented_list
+++ b/doc/interpreter/doccheck/mk_undocumented_list
@@ -82,32 +82,34 @@ chdir
 comma
 debug
 dbnext
 exit
 F_DUPFD
 F_GETFD
 F_GETFL
 finite
+fmod
 fntests
 F_SETFD
 F_SETFL
 gammaln
 home
 i
 ifelse
 inf
 inverse
 isbool
 isfinite
 j
 J
 lower
 lstat
 nan
+octave_tmp_file_name
 O_APPEND
 O_ASYNC
 O_CREAT
 O_EXCL
 O_NONBLOCK
 O_RDONLY
 O_RDWR
 O_SYNC
